manifests/deploy-kfp/openshift/base/add_resources.sh

```diff
@@ -3,9 +3,9 @@
 REPO=$1
 BRANCH=$2
 
-kustomize edit add resource ${REPO}//manifests/kustomize/base/installs/generic?ref=${BRANCH}
```

Comment:
did it work without the `//` ? iirc these were required, but maybe I'm wrong

Reply:
hm, I don't remember if I tested with it. Let me double check.